frontend/appflowy_flutter/lib/plugins/database/calendar/presentation/calendar_page.dart

```diff
@@ -168,6 +168,18 @@ class _CalendarPageState extends State<CalendarPage> {
                 }
               },
             ),
+            BlocListener<CalendarBloc, CalendarState>(
```

Comment:
optional: use `BlocConsumer` to replace the `BlocListener` and `BlocBuilder`

Reply:
This BlocListener is in a `MultiBlocListener`. If I make it a consumer, it might seem like this particular listener is special in some way.